finalPrep/annotations.js

```diff
@@ -2,41 +2,64 @@
 // Use the double // to create a new comment
 
 //Build a Bear
+// This is a function with parameters.
 function buildABear(name, age, fur, clothes, specialPower) {
+  // Create a variable with a string that uses interpolation.
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+//  Create a variable with an array.
   var demographics = [name, age];
+  // Create a variable with a string that uses concatenation.
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+  // Create an object.
   var builtBear = {
+    // Key/value pair
     basicInfo: demographics,
+    // Key/value pair
     clothes: clothes,
+    // Key/value pair
     exterior: fur,
+    // Key/value pair with a number
     cost: 49.99,
+    // Key/value pair with an array
     sayings: [greeting, powerSaying, "Goodnight my friend!"],
+    // Key/value pair with boolean.
     isCuddly: true,
   };
-
+// Returns values from the object.
   return builtBear
 }
-
+// Calls the function and passes arguments.
 buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
+// Calls the function and passes arguments.
 buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');
 
 
 
 //FizzBuzz
+// Defines a function with parameters
 function fizzBuzz(num1, num2, range) {
+  // Creates a loop.
```

Comment:
You could be a little more specific here by describing how the loop works. For example, `i <= range` specifies the number of times the loop will run based on the parameter passed in for the range argument.